### PR TITLE
Service: Robot Chassis

### PR DESCRIPTION
**Pull request by pelikhan**
This pull request defines a new service.

### CODE REVIEW

File: services/robotchassis.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+	extends: _sensor
+
+## Registers
+
+    ro position @ reading {
+	    x: i16.16 cm
+		y: i16.16 cm
+    	vx: i16.16 cm/s
+        vy: i16.16 cm/s
+		heading: i16.16 °
```
Comment (mmoskal) [inline services/robotchassis.md:15]: indentation seems off

File: services/robotchassis.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+	    x: i16.16 cm
+		y: i16.16 cm
+    	vx: i16.16 cm/s
+        vy: i16.16 cm/s
+		heading: i16.16 °		
+    }
+    
+The current position and orientation of the robot.
+
+    rw enabled: bool @ intensity
```
Comment (mmoskal) [inline services/robotchassis.md:20]: does it belong here? I think either we have a way to control position, or no control over motors at all.

File: services/robotchassis.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+
+	identifier: 0x19f4d06b
+	extends: _sensor
+
+## Registers
+
+    ro kinematics @ reading {
+        x: i16.16 cm
+        y: i16.16 cm
+	vx: i16.16 cm/s
```
Comment (mmoskal) [inline services/robotchassis.md:13]: indent still off

**Comment by pelikhan** [comment]
Aaargh!

________________________________

**Review by mmoskal — APPROVED** [review]

**Pull request merged**